extensions/2.0/Vendor/EXT_feature_metadata/0.0.0/README.md

```diff
@@ -0,0 +1,665 @@
+          "properties": {
+            "elevation": {
+              "name": "Elevation (m)",
+              "description": "Elevation of the terrain in scene (per-texel)",
+              "type": "UINT8",
+              "normalized": true
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Feature Tables
+
+![Feature Table Diagram](figures/feature-table.png)
+
+A **feature table** stores metadata for a class in a parallel array format. Each array is called a **property array** and corresponds to a class property. The data contained with a property array must match the data type of the class property. Furthermore, the set of property arrays must match one-to-one with the class properties.
+
+Feature tables in this extension conform to the definition of "instance table" from the  [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/0.0.0/README.md#instance-tables) except this extension uses slightly different terminology:
+
+| `EXT_feature_metadata` Terminology | Equivalent Cesium 3D Metadata Terminology |
+|---------|------|
+| Feature Table | Instance Table |
+| Feature ID | Instance ID |
+
+The Cesium 3D Metadata Specification allows two possible encodings for feature tables: JSON and binary. However, this glTF extension **requires** [binary encoding](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/0.0.0/README.md#binary-encoding). Requiring binary encoding enforces best practices for runtime performance.
+
+The binary encoding requires buffer views for storing the property arrays. glTF `bufferView`s fulfill this requirement exactly.
+
+Like classes, feature tables are defined in the root extension object in the glTF model. See the example below:
+
+```json
+{
+  "extensions": {
+    "EXT_feature_metadata": {
+      "classes": {
+        "tree": {
+          "properties": {
+            "height": {
+              "name": "Height (m)",
+              "description": "Height of tree measured from ground level",
+              "type": "FLOAT32"
+            },
+            "birdCount": {
+              "name": "Bird Count",
+              "description": "Number of birds perching on the tree",
+              "type": "UINT8"
+            },
+            "species": {
+              "name": "Species",
+              "description": "Species of the tree",
+              "type": "STRING"
+            }
+          }
+        }
+      },
+      "featureTables": {
+        "treeTable": {
+          "class": "tree",
+          "properties": {
+            "height": {
+              "bufferView": 3
+            },
+            "birdCount": {
+              "bufferView": 4
+            },
+            "species": {
+              "bufferView": 5,
+              "offsetBufferViews": [6]
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Feature Textures
+
+![Feature Texture Example](figures/feature-texture.jpg)
```

Comment:
This diagram is a bit too large

Reply:
Yeah. Should I resize the image (e.g. with an `<img>` tag with `width` and `height`

Reply:
Yes, that should be fine